tutorials/hist/NormalizeHistogram.C

```diff
@@ -0,0 +1,73 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 2 different methods of normalizing Histograms
+/// are shown, each with the original Histogram
+/// next to the normalized one.
+/// \macro_image
+/// \macro_code
+///
+/// The methods are:
+/// 1:
+/// ```
+/// hist2->Scale(1. / hist2->Integral());
+///```
+/// 2:
+/// ```
+///  hist3->Scale(1. / hist3->Integral("width"));
+/// ```
+///
+/// \author Advait Dhingra
+
+#include "TH2F.h"
+#include "TRandom.h"
+#include "TCanvas.h"
+
+void NormalizeHistogram()
+{
+
+   // all of the histograms
+
+   TH1F *hist1 = new TH1F("Unnormalized Histogram", "Unnormalized Hist", 100, 0, 100);
+   hist1->SetStats(0);
+   TH1F *hist2 = new TH1F("Method 1 Normalized", "Method 1 Normalized", 100, 0, 100);
+   hist2->SetStats(0);
+   TH1F *hist3 = new TH1F("Method 2 Normalized", "Method 2 Normalized", 100, 0, 100);
```

Comment:
If you use bin width == 1 then there's no difference between the normalizations with and without `"width"` :-) I'd recommend to use:
```suggestion
   std::array<double, 6> binsx{0, 5, 10, 20, 50, 100};
   TH1F *hist1 = new TH1F("Unnormalized Histogram", "Unnormalized Hist", binsx.size() - 1, &binsx[0]);
   hist1->SetStats(0);
   TH1F *hist2 = new TH1F("Method 1 Normalized", "Method 1 Normalized", binsx.size() - 1, &binsx[0]);
   hist2->SetStats(0);
   TH1F *hist3 = new TH1F("Method 2 Normalized", "Method 2 Normalized", binsx.size() - 1, &binsx[0]);
```